lisa/sut_orchestrator/azure/features.py

```diff
@@ -668,6 +669,168 @@ def _initialize(self, *args: Any, **kwargs: Any) -> None:
             len(all_nics) - self.origin_extra_synthetic_nics_count - 1
         )
 
+    # Subroutine for applying route table to subnet.
+    # We don't want to retry the entire routine if we catch an exception in this section.
+    @retry(HttpResponseError, tries=5, delay=1, backoff=1.3)
+    def _do_update_subnet(
```

Comment:
put private methods behind public methods.

Reply:
I don't understand, you want the private methods declared after the public ones?
I had put them closer to the top since _initialize was at the top, I just noticed there are other private methods closer to the end of the class. I'll move down there.

Reply:
moved :)

Reply:
`_initialize` should be moved to lower as well. But it's ok to keep legacy style, if it's confusing.